src/Controller/Component/EmailComponent.php

```diff
@@ -84,6 +84,9 @@ public function sendEventRejected(Contact $contact, Event $event)
             The following event you submitted to the Dallas Makerspace Calendar has been rejected.<br/><br/>
             <b>Event: </b>$event->name <br/>
             <b>Reason: </b>$rejectionReason <br/><br/>
+            Please do not reply to this email with questions regarding this event.  For further assistance use our <a
+                href="https://talk.dallasmakerspace.org">Talk Forum</a>
+            <br><br>
```

Comment:
It probably makes more sense for this to be part of the footer section than here.
Further, if we do not want emails, we should probably make this sent from a noreply address as well.

Reply:
I wanted the reference to Talk to be more aggressive than sticking it in the footer.  I like the idea of a noreply address and have added that change.